src/js/actions/transform.js

```diff
@@ -231,7 +235,40 @@ define(function (require, exports) {
         // Used to calculate the new top/left positions of all layers in relation to top/left
         // of the group
         var overallWidthRatio = size.w ? size.w / overallBounds.width : 0,
-            overallHeightRatio = size.h ? size.h / overallBounds.height : 0;
+            overallHeightRatio = size.h ? size.h / overallBounds.height : 0,
+            overallLeftOffset = 0,
+            overallTopOffset = 0,
+            direction;
+
+        if (size.hasOwnProperty("w")) {
```

Comment:
I see that maybe you were just following a pattern that exists below... so if you don't know the answer to this, ignore it.  It may be a hard-won lesson that just wasn't documented correctly.